src/test/java/de/tum/in/www1/artemis/FileIntegrationTest.java

```diff
@@ -107,176 +56,6 @@ void testGetTemplateFile() throws Exception {
-        assertThat(receivedIcon).isEqualTo("some data");
-    }
-
-    @Test
-    @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "INSTRUCTOR")
-    void testGetDragAndDropBackgroundFile() throws Exception {
-        Course course = courseUtilService.addEmptyCourse();
-        QuizExercise quizExercise = quizExerciseUtilService.createQuiz(course, ZonedDateTime.now(), null, QuizMode.SYNCHRONIZED);
-        DragAndDropQuestion dragAndDropQuestion = (DragAndDropQuestion) quizExercise.getQuizQuestions().get(1);
-        quizExerciseRepository.save(quizExercise);
-
-        MockMultipartFile file = new MockMultipartFile("file", "background.png", "application/json", "some data".getBytes());
-        JsonNode response = request.postWithMultipartFile("/api/fileUpload?keepFileName=false", file.getOriginalFilename(), "file", file, JsonNode.class, HttpStatus.CREATED);
-        String responsePath = response.get("path").asText();
-        String backgroundPath = fileService.manageFilesForUpdatedFilePath(null, responsePath, FilePathService.getDragAndDropBackgroundFilePath(), dragAndDropQuestion.getId());
-
-        dragAndDropQuestion.setBackgroundFilePath(backgroundPath);
-        courseRepo.save(course);
-        quizQuestionRepository.save(dragAndDropQuestion);
-
-        String receivedPath = request.get(backgroundPath, HttpStatus.OK, String.class);
-        assertThat(receivedPath).isEqualTo("some data");
-    }
-
-    @Test
-    @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "INSTRUCTOR")
-    void testGetDragItemFile() throws Exception {
-        Course course = courseUtilService.addEmptyCourse();
-        QuizExercise quizExercise = quizExerciseUtilService.createQuiz(course, ZonedDateTime.now(), null, QuizMode.SYNCHRONIZED);
-        DragAndDropQuestion dragAndDropQuestion = (DragAndDropQuestion) quizExercise.getQuizQuestions().get(1);
-        quizExerciseRepository.save(quizExercise);
-
-        DragItem dragItem = dragAndDropQuestion.getDragItems().get(0);
-        MockMultipartFile file = new MockMultipartFile("file", "background.png", "application/json", "some data".getBytes());
-        JsonNode response = request.postWithMultipartFile("/api/fileUpload?keepFileName=false", file.getOriginalFilename(), "file", file, JsonNode.class, HttpStatus.CREATED);
-        String responsePath = response.get("path").asText();
-        String dragItemPath = fileService.manageFilesForUpdatedFilePath(null, responsePath, FilePathService.getDragItemFilePath(), dragItem.getId());
-
-        dragItem.setPictureFilePath(dragItemPath);
-        courseRepo.save(course);
-        quizQuestionRepository.save(dragAndDropQuestion);
-
-        String receivedPath = request.get(dragItemPath, HttpStatus.OK, String.class);
-        assertThat(receivedPath).isEqualTo("some data");
-    }
-
-    @Test
-    @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "INSTRUCTOR")
-    void testGetFileUploadSubmission() throws Exception {
```

Comment:
Why were these tests removed (the methods are still there)？

Reply:
Because all of them use the old deprecated upload logic that I removed in this PR. I'll check the coverage once it runs through and see what I can do there. What makes sense.